### PR TITLE
Fixed inconsistent use of spdx: in RDF examples

### LINKED ISSUE #57: Typo:  2.1 - Fix inconsistent use of spdx: in RDF examples
For the RDF examples, some of the SPDX terms use the spdx: prefix, others use no prefix.

This was https://bugs.linuxfoundation.org/show_bug.cgi?id=1369

### PR DESCRIPTION
**Pull request by salicodes**
Fix #57 - Replaced http://spdx.org/rdf/terms# with spdx: and also rdf:comment with spdx:comment

### CODE REVIEW

File: chapters/2-document-creation-information.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Example:
 Example:
 
     <SpdxDocument rdf:about="http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82...">
-        <rdfs:comment>This document was created using SPDX 2.0 using licenses from the web site.</rdfs:comment>
```
Comment (goneall) [inline chapters/2-document-creation-information.md]: The ```rdfs:comment``` is actually correct.   The ```comment``` property is not part of the SPDX namespace, it is part of the [rdfs namespace](https://www.w3.org/TR/rdf-schema/).

File: chapters/2-document-creation-information.md [hunk 2]
```diff
@@ -315,7 +315,7 @@ Example:
 
 **2.10.4** Data Format: Free form text that can span multiple lines.
 
-In `tag:value` format this is delimited by `<text> .. </text>`, in RDF, it is delimited by `<rdfs:comment>`.
```
Comment (goneall) [inline chapters/2-document-creation-information.md]: Same comment as above - I won't comment on all of the changes, just revert the changes for ```rdfs:comment```

File: chapters/3-package-information.md [hunk 3]
```diff
@@ -405,11 +405,11 @@ Example:
     </Package>
 
     <Package rdf:about="...">
-        <downloadLocation rdf:resource="http://spdx.org/rdf/terms#noassertion"/>
+        <downloadLocation rdf:resource="spdx:noassertion"/>
```
Comment (goneall) [inline chapters/3-package-information.md]: This is correct - these are the changes will improve the consistency in the documentation

**Comment by salicodes** [comment]
No, the package-lock.json isn't intended to be part of the PR. I have been trying to remove that, please guide me @goneall

**Comment by goneall** [comment]
@salicodes I'm not familiar with the package-lock.json file, but from what I read on the web it looks like we should not git ignore the file.  To avoid this issue in the future, do a git status prior to committing and pushing the changes and only add the files you wish to add.  There are probably other steps to avoid changes to the package-lock.json file, but I'm not familiar with this function and can't provide much help.

To recover from the extra added file, I found this on stack overflow: https://stackoverflow.com/questions/12481639/remove-files-from-git-commit

Note you will have to do a push -f (force) to update the changes.

Another (messier) approach would be to copy back the original file and add a commit to the pull request putting the file back to its original state.

**Comment by salicodes** [comment]
@goneall , the solution works perfectly :+1:

**Comment by tsteenbe** [comment]
Could not do a clean merge of this PR but merged it in now, see https://github.com/spdx/spdx-spec/commit/a227cea17556abe1f067cc248699371439329012

**Pull request closed without merging**